cmake/HPX_AddExecutable.cmake

```diff
@@ -121,7 +121,9 @@ macro(add_hpx_executable name)
       get_filename_component(extension ${source} EXT)
       if(${extension} STREQUAL ".cu")
         message(${extension})
-        SET_SOURCE_FILES_PROPERTIES(${source} PROPERTIES LANGUAGE CXX)
+        SET_SOURCE_FILES_PROPERTIES(${source} PROPERTIES
+          LANGUAGE CXX
+          COMPILE_FLAGS ${HPX_CUDA_CLANG_FLAGS})
       endif()
```

Comment:
Shouldn't this be added to `hpx_setup_target` instead. The user might be interested in using CUDA code from within a shared or static library...